test/quantization/test_quantize_script.py

```diff
@@ -1521,6 +1521,7 @@ def forward(self, x):
                 x = F.upsample(x, (32, 32))
                 x = F.upsample_bilinear(x, (32, 32))
                 x = F.upsample_nearest(x, (32, 32))
+                x = torch.clamp(x, -2, 2)
```

Comment:
could you add tensor clamp as well, searching for clamp in pytorch doc shows:
torch.clamp (Python function, in torch)
torch.Tensor.clamp (Python method, in torch.Tensor)
torch.Tensor.clamp_ (Python method, in torch.Tensor)